model/manifest.go

```diff
@@ -130,6 +130,9 @@ type Manifest struct {
 	// A description of what your plugin is and does.
 	Description string `json:"description,omitempty" yaml:"description,omitempty"`
 
+	// A file path in the bundle that points to the plugins svg icon.
```

Comment:
@hanzei 
would it be worth mentioning that this should be a relative path and relative path from where? i.e. from the root or from the where the manifest resides?

Reply:
Added a note that this should be a relative path.

The manifest has to be in the root of the bundle. Hence, I don't a more accurate way to describe where the file should be.

Reply:
@hanzei 

`// A relative file path in the bundle that points to the plugins svg icon. This should be relative to the root of your bundle and the location of the manifest file.`

Just following the same convention that we have for `bundle_path`: https://github.com/mattermost/mattermost-server/blob/master/model/manifest.go#L184

Reply:
Fixed :+1: